src/WebJobs.Script.Scaling/IWorkerTable.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    public interface IWorkerTable
+    {
+        Task<IEnumerable<IWorkerInfo>> List();
+
+        Task AddOrUpdate(IWorkerInfo info);
+
+        Task Delete(IWorkerInfo info);
```

Comment:
Please add comments. e.g. it's not clear to me how Delete() is different from IScaleHandler.ScaleIn. Probably missing the big picture about what the interfaces are for, who implements then and who calls them.